azure-pipelines-dependency-graph-testing.yml

```diff
@@ -314,6 +314,14 @@ stages:
 
             displayName: 'dgtest'
 
+          - script: |
+              # These are tests that are missed by the dgtest dependency graph.
+              # In order to ensure coverage, we run them during each CI/CD cycle.
+              pytest \
+                tests/test_deprecation.py
```

Comment:
Placing the filenames in this pipeline YAML but let me know if we should move to an external file. I think this is okay for the time being (we can expand if this list grows significantly)